extras/module-node/duk_module_node.c

```diff
@@ -12,7 +12,7 @@ static duk_int_t duk__eval_module_source(duk_context *ctx, void *udata);
 #else
 static duk_int_t duk__eval_module_source(duk_context *ctx);
 #endif
-static void duk__push_module_object(duk_context *ctx, const char *id);
+static void duk__push_module_object(duk_context *ctx, const char *id, int main);
```

Comment:
Oops, this was never fixed.  `main` argument should be `duk_bool_t` for clarity right?